### ISSUE TITLE
[Feature]: Allow ShortcutKeyDescription to be Set on CoreWebView2ContextMenuItem

### ISSUE THREAD

**Issue by NeonBits** [MicrosoftEdge/WebView2Feedback#4727]
### Describe the feature/enhancement you need

I'm currently working on a WPF application where I need to override the context menu. 
This has been fairly straightforward so far, but I cannot work out how to specify a keyboard shortcut to be displayed on the menu item.

I would the shortcuts to appear the same as the default menu:
![image](https://github.com/user-attachments/assets/8b2e2f58-d8aa-410f-8d22-a34380486fc4)

The CoreWebView2ContextMenuItem.ShortcutKeyDescription property is read only.
The CoreWebView2ContextMenuItem.Label property documentation suggests that it "Will contain an ampersand for characters to be used as keyboard accelerator."

I've tried eg. "MyCommandLabel&Ctrl+H" but the shortcut is just incorporated into the label.

Am I missing something? Is it done this way due to localization?
It would be much easier to be able to override the property value with Set.

### The scenario/use case where you would use this feature

It's important for accessibility considerations and for "power users" to be able to specify the keyboard capabilities of the given commands,

### How important is this request to you?

Impactful. My app's user experience would be significantly compromised without it.

**Comment by ajtruckle** [comment]
I asked for this myself. They won't do it sadly. Their justification is that you can use a native menu instead of the WV2 menu. 

Whilst a native menu will render shortcuts correctly, the thing is that I want to use a WV2 menu. And, I can't for the life of me understand why the API does not let us set the short description on the right.

**Comment by ajtruckle** [comment]
https://github.com/MicrosoftEdge/WebView2Feedback/issues/2373

**Comment by ajtruckle** [comment]
@NeonBits I agree, if more users express a desire for this to be available for the WV2 API ... 😊👍 Happy coding! 🖖

**Comment by Mendrejk** [comment]
This is mind boggling. I'd say this is a basic use case of context menus. This way you are pretty much always forced to draw your own menus.
Are there any plans to support shortcuts for ICoreWebView2ContextMenuItems?
